chat-plugins/info.js

```diff
@@ -1821,10 +1822,10 @@ var commands = exports.commands = {
 			}
 			return this.sendReplyBox("Random number " + num + "x(1 - " + faces + "): " + rolls.join(", ") + "<br />Total: " + total);
 		}
-		if (target && isNaN(target) || target.length > 21) return this.sendReply("The max roll must be a number under 21 digits.");
```

Comment:
<code>target</code> should be casted to a number before this.
Just use <code>parseInt</code>. This will remove support for scientific notation in single-dice rolls, but it's not a big loss in lieu of standardizing the syntax for both basic and `ndx` notation, specially given that this one doesn't play nice with scientific notation.